### DISCUSSION TITLE
Please fix my 2FA

### DISCUSSION THREAD

**Discussion by fabian-flechtmann** [community/community#118581] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello github team,

i'm currently locked out of my account "fafl" (short for my first and last name). I don't have the phone anymore that I used to setup 2FA. I also don't have SSH keys set up and didn't download any recovery codes (smart of me, i know).

I still know the password, have access to my email and can read SMS messages.

Would it be possible to enable account recovery via SMS or email or to completely disable 2FA?

I've been trying to access this account since over a year. You will see no recent logins, only failures.

Thanks in advance
Fabian

**Comment by ebndev** [comment]
Hi [fabian-flechtmann](https://github.com/fabian-flechtmann)! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
